packages/document/module-doc/docs/en/api/config/build-config.mdx

```diff
@@ -67,12 +67,29 @@ export default {
 
 Contains configuration related to static assets.
 
-## asset.path
+## asset.name
 
-Static resource output path, will be based on [outDir](/api/config/build-config#outDir)
+Static resource output file name.
 
-- **Type**: `string`
-- **Default**: `assets`
+- **Type**: `string | ((assetPath) => name)`
+- **Default**: `[name].[hash].[ext]`
+
+When asset.name is a string, it will automatically replace [name], [ext], and [hash], respectively replaced by the file name, extension, and file hash.
```

Comment:
can we support custom hash length like webpack? such as `[hash:8]`

Reply:
OK, will do it